ansible/roles/sink-ssh-gateway/tasks/main.yml

```diff
@@ -1,7 +1,4 @@
 ---
-- hosts: tag_Name_cockpit_sink
-  tasks:
-
   - name: Create sshgw user
     user:
       name: sshgw
```

Comment:
Oops, I should have outdented this file for consistency. It apparently seems to work anyway, though.

Reply:
Fixed